RHEL6 - e2e tests needing docker registry do not pass

### Version of Singularity:

```
3.5.2
```


### Actual behavior
```
-- FAIL: TestE2E/PAR/BUILD/from/Root/OrasURI (0.02s)
                        imgbuild.go:124: Running command "/usr/local/bin/singularity --debug build --force --sandbox /tmp/stest.490211207/build-from.261839283/sandbox oras://localhost:5000/oras_test_sif:latest"
                        imgbuild.go:124: 
                            "/usr/local/bin/singularity --debug build --force --sandbox /tmp/stest.490211207/build-from.261839283/sandbox oras://localhost:5000/oras_test_sif:latest" output:
                            ESC[0mDEBUG   ESC[0m[U=0,P=12666]      persistentPreRunE()           Singularity version: 3.5.2+141-g7742e82
                            ESC[0mDEBUG   ESC[0m[U=0,P=12666]      handleConfDir()               /root/.singularity already exists. Not creating.
                            ESC[0mDEBUG   ESC[0m[U=0,P=12666]      initCacheDir()                Creating cache directory: /tmp/stest.490211207/e2e-imgcache-484644214/cache
                            ESC[0mDEBUG   ESC[0m[U=0,P=12666]      initCacheDir()                Creating cache directory: /tmp/stest.490211207/e2e-imgcache-484644214/cache/library
                            ESC[0mDEBUG   ESC[0m[U=0,P=12666]      updateCacheSubdir()           Caching directory set to /tmp/stest.490211207/e2e-imgcache-484644214/cache/library
                            ESC[0mDEBUG   ESC[0m[U=0,P=12666]      initCacheDir()                Creating cache directory: /tmp/stest.490211207/e2e-imgcache-484644214/cache/oci-tmp
                            ESC[0mDEBUG   ESC[0m[U=0,P=12666]      updateCacheSubdir()           Caching directory set to /tmp/stest.490211207/e2e-imgcache-484644214/cache/oci-tmp
                            ESC[0mDEBUG   ESC[0m[U=0,P=12666]      initCacheDir()                Creating cache directory: /tmp/stest.490211207/e2e-imgcache-484644214/cache/oci
                            ESC[0mDEBUG   ESC[0m[U=0,P=12666]      updateCacheSubdir()           Caching directory set to /tmp/stest.490211207/e2e-imgcache-484644214/cache/oci
                            ESC[0mDEBUG   ESC[0m[U=0,P=12666]      initCacheDir()                Creating cache directory: /tmp/stest.490211207/e2e-imgcache-484644214/cache/net
                            ESC[0mDEBUG   ESC[0m[U=0,P=12666]      updateCacheSubdir()           Caching directory set to /tmp/stest.490211207/e2e-imgcache-484644214/cache/net
                            ESC[0mDEBUG   ESC[0m[U=0,P=12666]      initCacheDir()                Creating cache directory: /tmp/stest.490211207/e2e-imgcache-484644214/cache/shub
                            ESC[0mDEBUG   ESC[0m[U=0,P=12666]      updateCacheSubdir()           Caching directory set to /tmp/stest.490211207/e2e-imgcache-484644214/cache/shub
                            ESC[0mDEBUG   ESC[0m[U=0,P=12666]      initCacheDir()                Creating cache directory: /tmp/stest.490211207/e2e-imgcache-484644214/cache/oras
                            ESC[0mDEBUG   ESC[0m[U=0,P=12666]      updateCacheSubdir()           Caching directory set to /tmp/stest.490211207/e2e-imgcache-484644214/cache/oras
                            ESC[0mDEBUG   ESC[0m[U=0,P=12666]      newBundle()                   Created temporary directory "/tmp/bundle-temp-848590480" for the bundle
                            ESC[0mDEBUG   ESC[0m[U=0,P=12666]      newBundle()                   Created directory "/tmp/stest.490211207/build-from.261839283/rootfs-cdac5b0a-24f4-11ea-995a-080027ba603d" for the bundle
                            ESC[34mINFO    ESC[0m[U=0,P=12666]      Full()                        Starting build...
                            ESC[0mDEBUG   ESC[0m[U=0,P=12666]      Get()                         Getting container from registry using ORAS
                            ESC[0mDEBUG   ESC[0m[U=0,P=12666]      cleanUp()                     Cleaning up "/tmp/stest.490211207/build-from.261839283/rootfs-cdac5b0a-24f4-11ea-995a-080027ba603d" and "/tmp/bundle-temp-848590480"
                            ESC[31mFATAL   ESC[0m[U=0,P=12666]      runBuildLocal()               While performing build: conveyor failed to get: failed to get SHA of oras://localhost:5000/oras_test_sif:latest: while resolving reference: failed to do request: Head http://localhost:5000/v2/oras_test_sif/manifests/latest: dial tcp [::1]:5000: connect: connection refused
                            
                        imgbuild.go:124: got 255 as exit code and was expecting 0: exit status 255
                            waiting for command "/usr/local/bin/singularity --debug build --force --sandbox /tmp/stest.490211207/build-from.261839283/sandbox oras://localhost:5000/oras_test_sif:latest"
                            github.com/sylabs/singularity/e2e/internal/e2e.TestEnv.RunSingularity.func1
                                github.com/sylabs/singularity@/e2e/internal/e2e/singularitycmd.go:628
                            github.com/sylabs/singularity/e2e/internal/e2e.Privileged.func1
                                github.com/sylabs/singularity@/e2e/internal/e2e/priv_linux.go:53
                            testing.tRunner
                                testing/testing.go:909
                            runtime.goexit
                                runtime/asm_amd64.s:1357
```

```
--- FAIL: TestE2E/PAR/PUSH/oras/standard_SIF_push (0.04s)
                    push.go:123: Running command "/usr/local/bin/singularity --debug push /tmp/stest.490211207/test.sif oras://localhost:5000/standard_sif:test"
                    push.go:123: 
                        "/usr/local/bin/singularity --debug push /tmp/stest.490211207/test.sif oras://localhost:5000/standard_sif:test" output:
                        ESC[0mDEBUG   ESC[0m[U=500,P=23747]    persistentPreRunE()           Singularity version: 3.5.2+141-g7742e82
                        ESC[0mDEBUG   ESC[0m[U=500,P=23747]    handleConfDir()               /home/dtrudg/.singularity already exists. Not creating.
                        ESC[0mDEBUG   ESC[0m[U=500,P=23747]    Init()                        Image format detection
                        ESC[0mDEBUG   ESC[0m[U=500,P=23747]    Init()                        Check for sandbox image format
                        ESC[0mDEBUG   ESC[0m[U=500,P=23747]    Init()                        sandbox format initializer returned: not a directory image
                        ESC[0mDEBUG   ESC[0m[U=500,P=23747]    Init()                        Check for sif image format
                        ESC[0mDEBUG   ESC[0m[U=500,P=23747]    Init()                        sif image format detected
                        ESC[31mFATAL   ESC[0m[U=500,P=23747]    func49()                      Unable to push image to oci registry: unable to push: failed to do request: Head http://localhost:5000/v2/standard_sif/blobs/sha256:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855: dial tcp [::1]:5000: connect: connection refused
                        
                    push.go:123: got 255 as exit code and was expecting 0: exit status 255
                        waiting for command "/usr/local/bin/singularity --debug push /tmp/stest.490211207/test.sif oras://localhost:5000/standard_sif:test"
                        github.com/sylabs/singularity/e2e/internal/e2e.TestEnv.RunSingularity.func1
                                github.com/sylabs/singularity@/e2e/internal/e2e/singularitycmd.go:628
                        testing.tRunner
                                testing/testing.go:909
                        runtime.goexit
                                runtime/asm_amd64.s:1357
```

### What OS/distro are you running

```
RHEL6
```

--- a/e2e/internal/e2e/docker.go
+++ b/e2e/internal/e2e/docker.go
@@ -35,7 +35,7 @@ func PrepRegistry(t *testing.T, env TestEnv) {
 		EnsureImage(t, env)
 
 		dockerDefinition := "testdata/Docker_registry.def"
-		dockerImage := filepath.Join(env.TestDir, "docker-e2e.sif")
+		dockerImage := filepath.Join(env.TestDir, "docker-registry")
 
 		env.RunSingularity(
 			t,
@@ -51,7 +51,7 @@ func PrepRegistry(t *testing.T, env TestEnv) {
 			t,
 			WithProfile(RootProfile),
 			WithCommand("instance start"),
-			WithArgs("-w", "-B", "/sys", dockerImage, dockerInstanceName),
+			WithArgs("-w", dockerImage, dockerInstanceName),
 			PreRun(func(t *testing.T) {
 				umountFn = shadowInstanceDirectory(t, env)
 			}),

--- a/e2e/testdata/Docker_registry.def
+++ b/e2e/testdata/Docker_registry.def
@@ -2,34 +2,18 @@ bootstrap: docker
 from: registry:2.7.1
 
 %post
-    apk add docker
-    # prevent docker to add iptables rules because it
-    # add few rules even with --iptables=false
-    rm -f /sbin/iptables
+    apk add runc --repository=http://dl-cdn.alpinelinux.org/alpine/edge/community \
+            img --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
 
 %startscript
-    # if there is no docker0 bridge, we could safely
-    # remove the created one after the docker daemon
-    # started
-    if ! brctl show docker0; then
-        DELETE_BRIDGE=1
-    fi
-
-    dockerd --iptables=false --ip-forward=false --ip-masq=false --storage-driver=vfs &
     /.singularity.d/runscript &
 
     # wait until docker registry is up
     while ! wget -q -O /dev/null 127.0.0.1:5000 ; do sleep 0.5; done
 
-    # delete bridge if docker0 was not previously present
-    if [ ! -z "${DELETE_BRIDGE}" ]; then
-        ip link set docker0 down || true
-        brctl delbr docker0 || true
-    fi
-
-    docker pull busybox || kill -TERM 1
-    docker tag busybox localhost:5000/my-busybox || kill -TERM 1
-    docker push localhost:5000/my-busybox || kill -TERM 1
+    img pull busybox || kill -TERM 1
+    img tag busybox localhost:5000/my-busybox || kill -TERM 1
+    img push localhost:5000/my-busybox || kill -TERM 1
 
     # e2e PrepRegistry will repeatedly trying to connect to this port
     # giving indication that it can start

